NST/src/main/java/com/ebay/jsonpath/JPIntegerCheck.java

```diff
@@ -59,14 +71,18 @@ public void processJsonPath(String jsonPath, SoftAssert softAssert, DocumentCont
       return;
     }
 
-    softAssert.assertNotNull(value, AssertMessageBuilder.build(jsonPath, "with null integer."));
+    if (isNull) {
```

Comment:
Shouldn't this be isNullExpected ? @byarger-ebay

Reply:
Yes, good catch. Thanks!